doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -755,19 +761,22 @@ Libraries for networking
     * Kconfig option :kconfig:option:`CONFIG_NRF_CLOUD_FOTA_AUTO_START_JOB` for controlling whether a FOTA update job is started automatically or at the request of the application.
     * An event :c:enum:`NRF_CLOUD_EVT_FOTA_JOB_AVAILABLE` that indicates a FOTA update job is available.
     * :c:func:`nrf_cloud_fota_job_start` function that starts a FOTA update job.
+    * :c:func:`nrf_cloud_shadow_delta_response_encode()` to help accept or reject shadow delta desired settings.
 
   * Updated:
 
     * JSON manipulation moved from :file:`nrf_cloud_fota.c` to :file:`nrf_cloud_codec_internal.c`.
     * :c:func:`nrf_cloud_obj_location_request_create` to use the new function :c:func:`nrf_cloud_obj_location_request_payload_add`.
     * Retry handling for P-GPS data download errors to retry ``ECONNREFUSED`` errors.
     * By default, Wi-Fi location requests include only the MAC address and RSSI value.
+    * The shadow desired section for the config subsection is no longer deleted. Applications and samples should use the function :c:func:`nrf_cloud_shadow_delta_response_encode()` to prevent recurring deltas.
```

Comment:
```suggestion
    * The shadow desired section for the config subsection is no longer deleted.
      Applications and samples should use the function :c:func:`nrf_cloud_shadow_delta_response_encode()` to prevent recurring deltas.
```